settings/htsohm.sample.yaml

```diff
@@ -47,19 +46,23 @@ interactive_mode: 'on'
 selection_mode: 'manual'
 
 simulations:
+  directory: 'HTSOHM'
```

Comment:
directory doesn't need to be within simulations, since it isn't a simulation. If you move it above, it makes it easy to iterate over your simulations